### PR TITLE
make the expired page prettier

### PR DESCRIPTION
**Pull request by colearendt**
make expired license page prettier

also fix links to be relative universally (in case behind a proxy...)

### CODE REVIEW

File: helper/workbench-for-microsoft-azure-ml/landing/expired.htm [hunk 1]
```diff
@@ -1,4 +1,90 @@
...
+  outline: none;
+}
+</style>
+
+
+<div>
+
+<a href="./"><img src="./RStudio-Logo-Flat.png"></a>
+
+<h2>Waiting for service to start...</h2>
```
Comment (blairj09) [inline helper/workbench-for-microsoft-azure-ml/landing/expired.htm:78]: Do we want to leave this note about the service waiting to start in the case of an expired / invalid license?

Comment (colearendt) [reply helper/workbench-for-microsoft-azure-ml/landing/expired.htm:78]: 🙈 Good catch, sorry haha.

Comment (colearendt) [reply helper/workbench-for-microsoft-azure-ml/landing/expired.htm]: Fixed!

**Review by blairj09 — APPROVED** [review]
LGTM 😁

**Pull request merged**